stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -293,6 +393,77 @@ pub fn boot_nakamoto<'a>(
     peer
 }
 
+/// Make a peer and transition it into the Nakamoto epoch.
+/// The node needs to be stacking and it needs to vote for an aggregate key;
```

Comment:
Presumably, we will no longer be voting for aggregate public keys?

Reply:
Removed this function entirely in 41a723c56